packages/components/src/core/NavigationJumpTo/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { forwardRef, useEffect } from "react";
+import { toKebabCase } from "src/common/utils";
+import { NavigationJumpToExtraProps, StyledTab, StyledTabs } from "./style";
+import useInView from "./useIntersection";
+
+export interface NavigationJumpToProps extends NavigationJumpToExtraProps {
+  indicatorColor?:
+    | "primary"
+    | "error"
+    | "info"
+    | "success"
+    | "warning"
+    | "gray"
+    | "beta";
+  items: Array<{
+    title: string;
+    elementRef: React.MutableRefObject<HTMLElement | null>;
+  }>;
+}
+
+const NavigationJumpTo = forwardRef<HTMLButtonElement, NavigationJumpToProps>(
+  (props, ref): JSX.Element | null => {
+    const { items, indicatorColor, ...rest } = props;
+    const [value, setValue] = React.useState(0);
```

Comment:
maybe rename the variable to `firstTabIndexInview` for readability?

Reply:
Of course :D